docs/build.yml

```diff
@@ -3,8 +3,10 @@ dependencies:
 - python=3.7
 - graphviz
 - pip:
+  - --find-links https://sxjscience.github.io/KDD2020/
   - jupyter-sphinx>=0.2.2
-  - mxnet-cu100==1.6.0b20191013
+  - gluonnlp==0.8.1
+  - mxnet_cu101>=1.7.0b20200713, <2.0.0
```

Comment:
Do we think it would be wise to limit to <1.8.0 ?

Reply:
I think versions less than 2.0.0 should be backward compatible so it should be fine.

Reply:
mxnet follows semantic versioning, so there shouldn't be any breakage in 1.x unless there's an issue in mxnet